.circleci/config.yml

```diff
@@ -234,29 +234,27 @@ jobs:
 
   reprotest-wheels:
     docker:
-      - image: circleci/python:3.7-buster
+      - image: quay.io/freedomofpress/packaging-debian-buster@sha256:5e1d3cf73ac6bfe418d499fc419bf789a0d365c51c157b704969d88d2df25e65
     steps:
       - checkout
       - run:
           name: install test requirements and run tests
           command: |
             make install-deps
-            virtualenv -p /usr/bin/python3 .venv
```

Comment:
where do we make the virtualenv then?

Reply:
The vitualenv creation was moved into the `install-deps` script.

https://github.com/freedomofpress/securedrop-debian-packaging/blob/main/scripts/install-deps#L43